src/components/ActionButton.tsx

```diff
@@ -16,9 +16,9 @@ export interface IActionButtonProps {
    */
   disabled?: boolean;
   /**
-   * Icon name
+   * Icon
    */
-  iconName: string;
+  icon: LabIcon.IMaybeResolvable;
```

Comment:
```suggestion
  icon: LabIcon;
```

All of the `ActionButton` instances are upgraded in this PR to actual `LabIcon` instances, so there's no need for `IMaybeResolvable` here.

In general, my advice would be to avoid `IMaybeResolvable` like the plague. Its use cases are very narrow

Reply:
Thanks for the tip